data.sql

```diff
@@ -62,3 +62,32 @@ update animals set owner_id = (select id from owners where full_name = 'Jennifer
 update animals set owner_id = (select id from owners where full_name = 'Bob') where name in ('Devimon', 'Plantmon');
 update animals set owner_id = (select id from owners where full_name = 'Melody Pond') where name in ('Charmander', 'Squirtle', 'Blossom');
 update animals set owner_id = (select id from owners where full_name = 'Dean Winchester') where name in ('Angemon', 'Boarmon');
+
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md) for this project, " `Insert the following data for vets:
Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008. `", please kindly insert the above data into your `vets` table.